ramls/loan-policy.json

```diff
@@ -110,6 +110,11 @@
               "$ref": "period.json",
               "description": "Recall return interval"
             },
+            "allowExtendOverdueLoanDueDate": {
```

Comment:
The story uses a specific natural language that I think reads quite well for a setting, `Allow recalls to extend overdue loans`.

Why was a different name chosen for the back end property? Is it to reduce duplication because it is already in a `recalls` object?

Reply:
Yes, I can add the `recalls` part.

Reply:
Changed to `allowRecallsExtendOverdueLoan`.

Reply:
Would it be ok to change it again to `allowRecallsToExtendOverdueLoans`?

Apologies :-/

Reply:
Sure, no prob.

Reply:
Thank you. Sorry :-(